pipeline/metadata/flatten_satellite.py

```diff
@@ -117,7 +116,13 @@ def _append_tags(tags: List[str]) -> str:
   Returns:
     string like "ip http cert"
   """
-  return ' '.join([tag for tag in tags if tag in SATELLITE_TAGS])
+  matches_control = MatchesControl(
+      ip='ip' in tags,
```

Comment:
This is not the right semantics. You should only set to false when the test actually happened.
CP is not checking IPs or AS when the AS information is missing, for instance.

For now, let's output True if present, and NULL otherwise.
If we add the control test to the pipeline, then we can correctly set the fields.

Reply:
BTW, one easy way to do that is to add the trusted answers as a domain tag.